grocery_story.R

```diff
@@ -1,6 +1,6 @@
 ## collect the variables, and pick one at random
 
-people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Miriam", "Linda", "Emmy", "Daniel")
+people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Miriam", "Linda", "Emmy", "Daniel",'Yinan')
```

Comment:
```suggestion
people <- c("Jennifer", "Albert", "Tiago", "Oisin", "Miriam", "Linda", "Emmy", "Daniel", "Yinan")
```
To keep the style of the original script, let's add:
- space between the names
- use " instead of '